src/css/Properties.js

```diff
@@ -435,6 +435,8 @@ var Properties = module.exports = {
     "text-decoration-color"             : "<text-decoration-color>",
     "text-decoration-line"              : "<text-decoration-line>",
     "text-decoration-style"             : "<text-decoration-style>",
+    "text-decoration-skip"              : "none | [ objects || spaces || ink || edges || box-decoration ]",
```

Comment:
The formal syntax in English differs from the version on the Russian translation.

```
none | [ objects || [ spaces | [ leading-spaces || trailing-spaces ] ] || edges || box-decoration ]
```

Can you check which is more up2date?

Reply:
Yes. I Update Formal Syntax. Get it in [Specification](https://drafts.csswg.org/css-text-decor-4/#text-decoration-skip-property)

Reply:
Okay, MDN finally confused me =)
There are [CSS Text Decoration Module Level 3](https://www.w3.org/TR/css-text-decor-3/#text-decoration-skip-property) [CR] That has formal syntax `none | [ objects || spaces || ink || edges || box-decoration ]`

And [CSS Text Decoration Module Level 4](https://drafts.csswg.org/css-text-decor-4/#text-decoration-skip-property) [ED] with formal syntax `none | [ objects || [ spaces | [ leading-spaces || trailing-spaces ] ] || edges || box-decoration ]`

I Think, it will be better to to use [CR] formal syntax instead [ED] version

Reply:
Ok, sounds good.